homeassistant/components/sensor/strings.json

```diff
@@ -160,6 +160,9 @@
     "energy_storage": {
       "name": "Stored energy"
     },
+    "enum": {
+      "name": "Enumerator"
```

Comment:
Or neither... Maybe just link sensor, it is a textual value.

Reply:
I think it does. `Enumeration` is not a thing you have. Device classes are used to indicate the type of device, like a temperature, humidity. Enumeration is more a meta, it is not like you have a nice enumuration in your living room... it would be just a sensor.